docs/quickstarts/rosa-osd-add-machine-pool /rosa-osd-add-machine-pool.yml

```
@@ -1,30 +1,31 @@
 
 metadata:
-  name: rosa-add-machine-pool
+  name: add-machine-pool
   instructional: true
 spec:
-  displayName: Add a machine pool to your ROSA cluster
+  displayName: Adding a machine pool to your managed OpenShift cluster
   durationMinutes: 5
   type:
     text: Quick start
     color: green
   icon: data:image/svg+xml;base64,PCEtLSBHZW5lcmF0ZWQgYnkgSWNvTW9vbi5pbyAtLT4KPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjUxMiIgaGVpZ2h0PSI1MTIiIHZpZXdCb3g9IjAgMCA1MTIgNTEyIj4KPHRpdGxlPjwvdGl0bGU+CjxnIGlkPSJpY29tb29uLWlnbm9yZSI+CjwvZz4KPHBhdGggZD0iTTQ0OCA2NHY0MTZoLTMzNmMtMjYuNTEzIDAtNDgtMjEuNDktNDgtNDhzMjEuNDg3LTQ4IDQ4LTQ4aDMwNHYtMzg0aC0zMjBjLTM1LjE5OSAwLTY0IDI4LjgtNjQgNjR2Mzg0YzAgMzUuMiAyOC44MDEgNjQgNjQgNjRoMzg0di00NDhoLTMyeiI+PC9wYXRoPgo8cGF0aCBkPSJNMTEyLjAyOCA0MTZ2MGMtMC4wMDkgMC4wMDEtMC4wMTkgMC0wLjAyOCAwLTguODM2IDAtMTYgNy4xNjMtMTYgMTZzNy4xNjQgMTYgMTYgMTZjMC4wMDkgMCAwLjAxOS0wLjAwMSAwLjAyOC0wLjAwMXYwLjAwMWgzMDMuOTQ1di0zMmgtMzAzLjk0NXoiPjwvcGF0aD4KPC9zdmc+Cg==
   prerequisites:
-    - You must have a ROSA cluster.
+    - You must have a managed Openshift cluster.
     - You must be a cluster owner, cluster editor, or Organization Administrator for the cluster.
   description: |-
-    Machine pools are an elastic, dynamic provisioning method used on top of your cloud infrastructure.
+    Add a machine pools to your managed OpenShift cluster.
```

```suggestion
    Add a machine pool to your managed OpenShift cluster.
```
Sorry- missed this earlier on.